integration_tests/fixtures/applicationData.json

```diff
@@ -319,22 +319,21 @@
     "alleged-offence-data": [
       {
         "titleAndNumber": "Arson",
-        "offenceCategory": "Arson",
         "offenceDate-day": "5",
         "offenceDate-month": "6",
-        "offenceDate-year": "1940",
-        "summary": "summary detail"
+        "offenceDate-year": "1940"
       },
       {
         "titleAndNumber": "Stalking",
-        "offenceCategory": "Stalking",
         "offenceDate-day": "6",
         "offenceDate-month": "7",
-        "offenceDate-year": "2023",
-        "summary": "more summary detail"
+        "offenceDate-year": "2023"
       }
     ],
-    "alleged-offences": {}
+    "alleged-offences": {},
```

Comment:
I guess the implication of this is that the task can be complete even without any alleged offences. Do you think we should add some validation to prevent that, given that there will always be an alleged offence?

Reply:
oh wait sorry, ignore me :)